src/main/menu/application-menu-items.injectable.ts

```diff
@@ -69,7 +69,8 @@ const applicationMenuItemsInjectable = getInjectable({
             label: `About ${productName}`,
             id: "about",
             click(menuItem: MenuItem, browserWindow: BrowserWindow) {
-              showAbout(browserWindow);
+              showAbout(browserWindow)
```

Comment:
Is there a reason why it's now promise which is not `await`ed? It's not good to left promises in the air. Those invite unwelcome complexity and should be avoided.